spec/controllers/provider_foreman_controller_spec.rb

```diff
@@ -95,7 +95,7 @@
     end
 
     it "renders explorer based on RBAC access to feature 'provider_foreman_add_provider'" do
-      set_user_privileges user_with_feature %w(provider_foreman_add_provider)
+      login_as user_with_feature %w(provider_foreman_add_provider)
```

Comment:
I've added what I mention above here in 9af8717 - see the commit message there for details.